src/diff/children.js

```diff
@@ -163,15 +163,24 @@ export function diffChildren(
 				firstChildDom = newDom;
 			}
 
-			oldDom = placeChild(
-				parentDom,
-				childVNode,
-				oldVNode,
-				oldChildren,
-				excessDomChildren,
-				newDom,
-				oldDom
-			);
+			if (childVNode._bailed) {
```

Comment:
It would be better if we were able to somehow move this away from the `vnode` and report it from the `diff` result, this would involve OR not returning `newDom` OR attaching it to the returned dom-node.

Scenario 1 makes the code above this one harder:
- looking for `firstChildDom`
- applying refs

Reply:
QQ: couldn't we check `childVNode === oldVNode` here instead of adding the _bailed flag?

Reply:
Found out we can use new_children === old_children thanks to https://github.com/preactjs/preact/blob/master/src/diff/index.js#L176